### PR TITLE
Semantic domains tests

### PR DESCRIPTION
**Pull request by JeanneSon**
## Description

The following test has been converted to playwright
✓ should be using English Semantic Domain for manager

The other tests had to be combined to one large tests called *Can change Project default language to Thai & back and forth* as they were all dependent on each other
✓ can change Project default language to Thai
✓ should be using Thai Semantic Domain
✓ can change Project default language back to English
✓ should be using English Semantic Domain
✓ can change Project default language back to Thai
✓ should be using Thai Semantic Domain after refresh
✓ can change user interface language
✓ should still have Thai for Project default language
✓ should be using English Semantic Domain
✓ should be using English Semantic Domain after refresh
✓ should still have Thai for Project default language
✓ can change user interface language to English
✓ can change Project default language to match interface language twice
✓ can change user interface language to back English

For the conversion,  a helper function was needed which allows the developer to check the selected option of
a select dropdown. It is located in the file *playwright-helpers.ts*.

### Type of Change
- E2E test

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the title/description of this PR which will be used as the squashed PR commit message
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: test/e2e/utils/playwright-helpers.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { expect, Locator, Page } from '@playwright/test';
```
Comment (JeanneSon) [inline test/e2e/utils/playwright-helpers.ts:1]: I am not sure yet whether the name of this file (playwright-helpers.ts) is good.
Do you have other suggestions?

Comment (longrunningprocess) [reply test/e2e/utils/playwright-helpers.ts:1]: seems fine to me for now, if we determine a better name later, we'll change it then.

**Review by longrunningprocess — CHANGES_REQUESTED** [review]
that was a lot, nice work!

I'm getting a failing test locally:
 1 failed
    [chromium] › semantic-domains.spec.ts:46:3 › Lexicon E2E Semantic Domains Lazy Load › Can change Project default language to Thai & back and forth 

```
Error: expect(received).toHaveText(expected)

- Expected string  - 1
+ Received string  + 3

- ภาษาไทย
+
+         
+         English
Call log:
  - expect.toHaveText with timeout 5000ms
  - waiting for selector "#languageDropdownButton"
  -   selector resolved to <a href="" aria-haspopup="true" aria-expanded="fals…>…</a>
  -   unexpected value "
        
        English"
  -   selector resolved to <a href="" aria-haspopup="true" aria-expanded="fals…>…</a>
  -   unexpected value "
        
        English"
  -   selector resolved to <a href="" aria-haspopup="true" aria-expanded="fals…>…</a>
  -   unexpected value "
        
        English"
  -   selector resolved to <a href="" aria-haspopup="true" aria-expanded="fals…>…</a>
  -   unexpected value "
        
        English"
  -   selector resolved to <a href="" aria-haspopup="true" aria-expanded="fals…>…</a>
  -   unexpected value "
        
        English"
  -   selector resolved to <a href="" aria-haspopup="true" aria-expanded="fals…>…</a>
  -   unexpected value "
        
        English"
  -   selector resolved to <a href="" aria-haspopup="true" aria-expanded="fals…>…</a>
  -   unexpected value "
        
        English"
  -   selector resolved to <a href="" aria-haspopup="true" aria-expanded="fals…>…</a>
  -   unexpected value "
        
        English"


  59 |     await projectSettingsPage.projectTab.saveButton.click();
  60 |     await expectOptionSelectedInSelectElement(projectSettingsPage.projectTab.defaultInterfaceLanguageInput, 'ภาษาไทย');
> 61 |     await expect(pageHeader.languageDropdownButton).toHaveText('ภาษาไทย');
     |                                                     ^
  62 |
  63 |     // should be using Thai semantic domain
  64 |     await editorPage.goto();

```

**Comment by megahirt** [comment]
All tests pass after rebasing...

**Review by longrunningprocess — APPROVED** [review]

**Pull request merged**